src/components/layout/header/platform-switcher/Data.tsx

```diff
@@ -0,0 +1,28 @@
+export type TPlatformCard = Record<'src' | 'title' | 'alt' | 'path', string>;
+
+export const available_apps: TPlatformCard[] = [
+    {
+        src: '/images/common/ic-rebranding-deriv-trader.svg',
+        title: 'A whole new trading experience on a powerful yet easy to use platform.',
+        alt: 'dtrader icon',
+        path: '/',
```

Comment:
Please add the correct redirection to these icons, they are suppose to redirect to app.deriv